dll/win32/shell32/lang/id-ID.rc

```diff
@@ -815,10 +812,10 @@ BEGIN
     IDS_FORMAT_COMPLETE "Format Selesai."
 
     /* Run File dialog */
-    IDS_RUNDLG_ERROR "Unable to display Run File dialog box (internal error)"
-    IDS_RUNDLG_BROWSE_ERROR "Unable to display Browse dialog box (internal error)"
-    IDS_RUNDLG_BROWSE_CAPTION "Browse"
-    IDS_RUNDLG_BROWSE_FILTER "Executable Files (*.exe)\0*.exe\0All Files (*.*)\0*.*\0"
+    IDS_RUNDLG_ERROR "Tidak bisa menampilkan kotak dialog Jalankan Berkas (kesalahan internal)"
+    IDS_RUNDLG_BROWSE_ERROR "Tidak bisa menampilkan kotak dialog Jelajah (kesalahan internal)"
+    IDS_RUNDLG_BROWSE_CAPTION "Jelajah"
+    IDS_RUNDLG_BROWSE_FILTER "Berkas Eksekusi (*.exe)\x00*.exe\x00Semua Berkas (*.*)\x00*.*\x00"
```

Comment:
Why using `\x00` instead of the regular `\0` ?

Reply:
i'm sorry, because i do copy-paste from resource hacker. fixed for now